generators/generator-adaptive-bot/generators/app/index.js

```diff
@@ -179,4 +220,20 @@ module.exports = class extends Generator {
             );
         }
     }
+
+    _writeRuntimeJson() {
+        const botName = this.options.botName;
+        const fileName = 'runtime.json';
```

Comment:
I assume we change this to appsettings.json and then line 231 to runtime.runtimeSettings.plugins.push(pluginDefinition)